react/src/containers/ChangePasswordModal.js

```diff
@@ -0,0 +1,91 @@
+import React, {useState} from 'react';
+import {Modal, Button, Form} from 'react-bootstrap';
+import * as Yup from 'yup';
+import {useFormik} from 'formik';
+import {useSelector, useDispatch} from 'react-redux';
+import {changePassword} from '../store/actions/UserActions';
+
+export default function ChangePasswordModal() {
+    const [show, setShow] = useState(false);
+    const user = useSelector(state => state.userReducer.userData.data);
```

Comment:
mozes napraviti ove selectore posebno, pogledaj kako smo to radili na https://github.com/Vivify-Ideas/expo-boilerplate/blob/master/store/selectors/UserSelector.js
Mozda to cak imas vec postavljeno na ovom boilerplate-u 💯

Reply:
Nema postavljeno, dodacu